src/app/plugins/platform/Docker/plugins/cms/Drupal8/index.ts

```diff
@@ -280,6 +281,7 @@ class Drupal8 extends Generator {
   writing() {
     const needsMemcached = this.options.plugins.cache === 'Memcache';
     const sharedDependencies = needsMemcached ? [memcached] : [];
+    sharedDependencies.push(xdebug);
```

Comment:
Let's change the logic of this conditional to make it clearer where the exceptional case is:

```ts
const sharedDependencies = [xdebug, opcache];
if (needsMemcached) {
  sharedDependencies.push(memcached);
}
```